### DISCUSSION TITLE
json error, not show meeting list on webpage where it was previously posted

### DISCUSSION THREAD

**Discussion by trivalleyaa** [code4recovery/12-step-meeting-list#1063] [General]
### Contact Details

valleyscaa@gmail.com

### Website With Issue

trivalleyaa.org/meetings

### What happened?

initially I went in to change my widgets, and one widget is for the Meeting Page top and botton where I had info posted with links,
I could not do anything in the entire widgets section, it was locked with lots of error messages along the top of the widget working page.Many mentioned Json error but I do not read code so did not know what it was about. I deleted the text in the Meeting Page Top box and the Meeting Page footer box. And deleted many other boxes in the Widget sections and was able to fix the sidebar widget. However, the TSML would not repopulate.
In addition, I am having a similar posting of TSML meetings on a new site I am building. The meetings show up on the Meeting Sidebar in the WP Dashboard, but not on a website page.
Any suggestions would be very helpful.

### What browsers are you seeing the problem on?

Firefox, Chrome, Safari

### Relevant log output/errors

```shell
Sorry, but I did have a json validation error for the second site. Json syntax error
```

**Comment by tim-rohrer** [comment]
I’m not fully following your description of your issue, so we may need a few back and forths.

First, I see your meetings here: https://trivalleyaa.org/?post_type=tsml_meeting

So your first concern is that you don’t see the meeting list, you may need to rebuild the link to it, or perhaps update the permalinks (General—>Permalinks—Save).

Any new instances of the Upcoming Meetings widget need to be created in a Wordpress Legacy Widget block. Try that and see if you’re able to get them back.
